test/testdata/generate.sh

```diff
@@ -113,10 +113,10 @@ function scaffold_test_project {
 
 build_kb
 
-# Project version 2 uses plugin go/v2 (default).
-scaffold_test_project project-v2 --project-version=2
-scaffold_test_project project-v2-multigroup --project-version=2
-scaffold_test_project project-v2-addon --project-version=2
+# Project version 2 uses plugin go/v2.
+scaffold_test_project project-v2 --plugins=go/v2 --project-version=2
+scaffold_test_project project-v2-multigroup --plugins=go/v2 --project-version=2
+scaffold_test_project project-v2-addon --plugins=go/v2 --project-version=2
```

Comment:
We cannot change here to ensure that it still working as before.

Reply:
There is a test in the unit test suite that checks that the behavior is the same. I think that being explicit in interation tests (both testdata related tests and e2e test) is better than relying on defaults.